lib/python/rose/macro.py

```diff
@@ -1378,7 +1382,7 @@ def apply_macro_to_config_map(config_map, meta_config, macro_function,
     """Apply a transform macro function to a config_map."""
     new_config_map = {}
     changes_map = {}
-    conf_keys = sorted(config_map)
+    conf_keys = list(config_map.keys())
```

Comment:
FYI this is equivalent to:
```python
list(config_map)
```

Reply:
Yes. I'll change it for consistency. It's an aspect of :snake: 3 that I quite like